### PR TITLE
Relocate JOHOR (VJB) VOR

### PR DESCRIPTION
**Pull request by Yifan081**
Fixed position, modified identifier.

### CODE REVIEW

File: Fixes.json [hunk 1]
```diff
@@ -39730,6 +39740,9 @@
     "Longitude": -89.11546
   },
   {
+    "Latitude": 0.0,
+    "Name": "ANUMA",
+    "Longitude": -10.0
```
Comment (Adrien-Z) [inline Fixes.json]: Correct the formatting by adding  _},  {_.

File: Fixes.json [hunk 2]
```diff
@@ -39730,7 +39740,12 @@
     "Longitude": -89.11546
   },
   {
-    "Latitude": 32.3374138,
+    "Latitude": 0.0,
```
Comment (Adrien-Z) [inline Fixes.json]: Insert the correct latitude for the fix (ANUMA) here.

File: Fixes.json [hunk 3]
```diff
@@ -39730,7 +39740,12 @@
...
+    "Name": "ANUMA",
+    "Longitude": -10.0
```
Comment (Adrien-Z) [inline Fixes.json]: Insert the correct longitude for the fix (ANUMA) here.

File: Fixes.json [hunk 4]
```diff
@@ -39730,7 +39740,12 @@
...
+    "Latitude": 32.3374138
```
Comment (Adrien-Z) [inline Fixes.json]: Delete that line.

**Comment by dushyants19** [comment]
Use navigation.airportediting.com thanks :)

**Pull request closed without merging**